banzai/main.py

```diff
@@ -314,31 +314,45 @@ def add_super_calibration():
     parser.add_argument('--db-address', dest='db_address',
                         default='mysql://cmccully:password@localhost/test',
                         help='Database address: Should be in SQLAlchemy form')
+    parser.add_argument('--skip-ingester', dest='skip_ingester', action='store_true')
+    parser.add_argument('--frame-id', dest='frame_id', help="Frame ID for this calibration frame, if sourced from the archive.")
     args = parser.parse_args()
     add_settings_to_context(args, settings)
     logs.set_log_level(args.log_level)
     frame_factory = import_utils.import_attribute(settings.FRAME_FACTORY)()
+
     try:
         cal_image = frame_factory.open({'path': args.filepath}, args)
     except Exception:
         logger.error(f"Calibration file not able to be opened by BANZAI. Aborting... {logs.format_exception()}",
                      extra_tags={'filename': args.filepath})
-        cal_image = None
+        return
+
+    if args.skip_ingester:
+        logger.debug("Skipped posting frame to archive. Saving to database.", extra_tags={'frameid': args.frame_id})
+        cal_image.frameid = args.frame_id
```

Comment:
If we aren't going to the ingester, the frame id isn't really meaningful (that is the archive frame id). Things should work if frame id is none (i.e. not in the archive) but the row has a filepath and a filename.